stubs/inertia-vue/resources/js/Components/DropdownLink.vue

```diff
@@ -1,5 +1,15 @@
 <template>
-    <inertia-link class="block w-full px-4 py-2 text-left text-sm leading-5 text-gray-700 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 transition duration-150 ease-in-out">
+    <Link class="block w-full px-4 py-2 text-left text-sm leading-5 text-gray-700 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 transition duration-150 ease-in-out">
```

Comment:
This is missing a href field. We will get an error like similar to this https://github.com/inertiajs/inertia/issues/744 id no href field on a Link.